src/java.sql.rowset/share/classes/com/sun/rowset/JoinRowSetImpl.java

```diff
@@ -633,10 +633,10 @@ private void initJOIN(CachedRowSet rowset) throws SQLException {
                 // Now we have two rowsets crsInternal and cRowset which need
                 // to be INNER JOIN'ED to form a new rowset
                 // Compare table1.MatchColumn1.value1 == { table2.MatchColumn2.value1
-                //                              ... upto table2.MatchColumn2.valueN }
+                //                              ... up to table2.MatchColumn2.valueN }
```

Comment:
Curious: it is not some established string representation, is it?

Reply:
You mean writing "upto" instead of "up to"? No, it's just incorrect, and a somewhat common typo.